bootstrap/main.go

```diff
@@ -38,6 +38,7 @@ import (
 
 	bmh_v1alpha1 "github.com/metal3-io/baremetal-operator/apis/metal3.io/v1alpha1"
 	metal3 "github.com/metal3-io/cluster-api-provider-metal3/api/v1beta1"
+	v1beta12 "github.com/openshift-assisted/cluster-api-agent/controlplane/api/v1beta1"
```

Comment:
should we consistently alias this imports to `controlplanev1beta1` and `bootstrapv1beta1` ?